README.md

```diff
@@ -10,6 +10,8 @@ See also `:help lsp-config`.
 
 ## Install
 
+[![LuaRocks](https://img.shields.io/luarocks/v/teto/nvim-lspconfig?logo=lua&color=purple)](https://luarocks.org/modules/teto/nvim-lspconfig)
```

Comment:
this is now released by the "neovim" user can you update the urls ? https://luarocks.org/modules/neovim/nvim-lspconfig 
then I'll merge